packages/eas-cli/src/commands/update/list.ts

```diff
@@ -0,0 +1,102 @@
+import { getConfig } from '@expo/config';
+import { Command, flags } from '@oclif/command';
+import Table from 'cli-table3';
+
+import { getUpdates } from '../../../src/update/updateUtils';
+import log from '../../log';
+import { ensureProjectExistsAsync } from '../../project/ensureProjectExists';
+import { findProjectRootAsync, getProjectAccountNameAsync } from '../../project/projectUtils';
+import { promptAsync } from '../../prompts';
+
+export default class UpdateList extends Command {
+  static hidden = true;
+  static description = 'View a list of updates.';
+
+  static args = [
+    {
+      name: 'releaseName',
+      required: false,
+      description: 'Name of the release to view',
+    },
+  ];
+
+  static flags = {
+    platform: flags.string({
+      description: 'Platform-specifc updates to return: ios, android, web.',
```

Comment:
```suggestion
      description: 'Platform-specifc updates to return: ios, android or both.',
```
(we don't support web now and won't in the near future either.)